### ISSUE TITLE
e2e test improvements

### ISSUE THREAD

**Issue by AMBudnik** [handsontable/handsontable#6491]
### Description

Here's a list of proposed changes for the e2e tests. I might be adding some more scenarios in the future and renaming it to an epic.

### `RowHeader.spec.js`
* build a test with `afterGetRowHeader` hook
* https://github.com/handsontable/handsontable/blob/master/test/e2e/RowHeader.spec.js#L49 fix typo. If we set up `const startRows = 5;` the scenario should be renamed to `should show row headers numbered 1-10 by default`

### `PluginHooks.spec.js`
* fix typos https://github.com/handsontable/handsontable/blob/master/test/e2e/PluginHooks.spec.js#L16

### `Core_setDataAtCell.spec.js`
* add a scenario for: Should allow using `setDataAtCell` when `editor:false`

**Comment by aninde** [comment]
Unlucky one e2e test does not pass on Firefox 71. The same test passed on an earlier version.
```
manualRowResize > handle position in a table positioned using CSS's `transform` > should display the handles in the correct position, with holder as a scroll parent
Expected 719 to be close to 720, 0.
```

**Comment by wszymanski** [comment]
I will also change the structure for the `HiddenColumns` plugin. I will make it similar to the structure of `HiddenRows` plugin.

**Comment by aninde** [comment]
This issue is done. The tests are included in `v8.1.0`.

**Comment by AMBudnik** [comment]
Thank you for considering my changes an adding them to v8.1.0. We can close this issue as done.
